googletest/test/gtest_unittest.cc

```diff
@@ -143,7 +143,37 @@ TEST_F(StreamingListenerTest, OnTestPartResult) {
 
   // Meta characters in the failure message should be properly escaped.
   EXPECT_EQ(
-      "event=TestPartResult&file=foo.cc&line=42&message=failed%3D%0A%26%25\n",
+      "event=TestPartResult&type=failure&file=foo.cc&line=42&message=failed%3D%0A%26%25\n",
+      *output());
+}
+
+TEST_F(StreamingListenerTest, OnTestPartResultSuccess) {
+  *output() = "";
```

Comment:
Some formating required. Check previous TEST_F invocations.

Reply:
I tried to mode this one on the one above:
https://github.com/google/googletest/blob/459bd19cdf77b3c73dbdd308f08d002aaa46b1ea/googletest/test/gtest_unittest.cc#L139

I checked again and don't see a formatting difference, but maybe I'm missing something.